app/experimenter/nimbus-ui/public/index.html

```diff
@@ -1,20 +1,5 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="description" content="" />
-    <meta name="referrer" content="origin" />
-    <meta name="robots" content="noindex,nofollow" />
-    <meta
-      name="viewport"
-      content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=2,user-scalable=yes"
-    />
-    <title>Mozilla Experimenter</title>
-  </head>
-  <body>
-    <noscript>Mozilla Experimenter requires JavaScript to run.</noscript>
-    <div id="root"></div>
-  </body>
-</html>
+<!--
+  NOTICE: This HTML file is no longer used. To modify the HTML file used
+  for this React app please make changes to the following file:
+  app/experimenter/legacy-ui/templates/experiments/nimbus-ui.html
+-->
```

Comment:
I like the idea of leaving this around with a comment👍

Reply:
Also CRA blows up if it's not present :)